### PR TITLE
Fix title tag hover in scaladoc

### PR DESCRIPTION
**Pull request by Philippus**
as can be seen on f.e. this page:
http://www.scala-lang.org/files/archive/api/current/scala/beans/BeanInfoSkip.html
when hovering over the links on the right the title tags contain line breaks and spaces in unwanted places.
Additionally two link elements were spread out over multiple lines

### CODE REVIEW

**Comment by Philippus** [comment]
@janekdb is this PR something for you to review?

File: src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala [hunk 1]
```diff
@@ -90,15 +90,13 @@ trait EntityPage extends HtmlPage {
                           mbr match {
                             case dtpl: DocTemplateEntity =>
                               dtpl.companion.fold(<span class="separator"></span>) { c: DocTemplateEntity =>
-                                <a class="object" href={relativeLinkTo(c)} title={c.comment.fold("")(com => Page.inlineToStr(com.short))}></a>
+                                <a class="object" href={relativeLinkTo(c)} title={c.comment.fold("")(com => Page.inlineToStrForTitleTag(com.short))}></a>
```
Comment (adriaanm) [inline src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala]: Could we lift `_.fold("")(com => Page.inlineToStrForTitleTag(com.short))` to a method? It's repeated at least three times.

Comment (Philippus) [reply src/scaladoc/scala/tools/nsc/doc/html/page/Entity.scala]: Yes, I was contemplating this as well. Will make it part of this PR.

**Review by adriaanm — APPROVED** [review]
Great, thanks!

**Pull request merged**